instrumentation/github.com/tidwall/buntdb/splunkbuntdb/example_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunkbuntdb_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	buntdb "github.com/signalfx/signalfx-go-tracing/contrib/tidwall/buntdb/splunkbuntdb"
```

Comment:
Please do not alias this. Leaving it as `splunkbuntdb` will clearly communicate in the example where the `splunkbuntdb` is actually used.

```suggestion
	"github.com/signalfx/signalfx-go-tracing/contrib/tidwall/buntdb/splunkbuntdb"
```